Comp2022/src/main/cpp/subsystems/Vision.cpp

```diff
@@ -28,12 +31,33 @@ Vision::Vision()
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
 
     table = nt::NetworkTableInstance::GetDefault().GetTable("limelight");
-    table->PutNumber("ledMode", LED_CUR_MODE);
+    table->PutNumber("ledMode", LED_ON);
     table->PutNumber("stream", PIP_SECONDARY);
 
     Initialize();
 }
 
+double Vision::CalculateDist()
+{
+    frc2135::RobotConfig *config = frc2135::RobotConfig::GetInstance();

```

Comment:
Eventually (doesn't need to be today) please move lines 42-46 into the Initialize function. We don't need to be re-reading the config every 20ms - only on startup

Would be especially cool if we could define a
```
void Vision::Readconfig()
{
    frc2135::RobotConfig *config = blah...
    blah blah blah
}
```
and then we can call this function in Vision::Initialize. We might decide we want to call this function at other times, too, so putting it into a function makes it easy to re-use